### DISCUSSION TITLE
Is this intended to be for PHP ^8.1 only?

### DISCUSSION THREAD

**Discussion by marcreichel** [spatie/laravel-login-link#3] [Q&A] [answered]
Hi, just wanted to install this package and stumbled upon the PHP ^8.1 requirement.
Is PHP ^8.1 really needed or could this be “downgraded” to ^8.0.2 just like Laravel itself, so this package could be used by more people (like me) who have not upgraded to PHP 8.1 yet?

**Comment by freekmurze** [comment, marked as answer]
I don't think we use PHP 8. 1 features, so I'll lower the requirement 👍
